include/seqan3/range/views/minimiser.hpp

```diff
@@ -171,6 +162,8 @@ class minimiser_view<urng_t>::window_iterator
     using it_t = std::ranges::iterator_t<rng_t>;
     //!\brief The sentinel type of the underlying range.
     using sentinel_t = std::ranges::sentinel_t<rng_t>;
+    //!\brief The sentinel type of the minimiser_view.
+    using sentinel_type = std::ranges::default_sentinel_t;
```

Comment:
One thing. I find it a little confusing now that there is a sentinel_t and a sentinel_type. Can we replace all instances with std::ranges::default_sentinel_t and omit this type definition here? 
It would make it less confusing.

Reply:
I agree, could you change this please? Or rename `sentinal_t` to `underlying_rng_sentinel_t`?

Reply:
I would prefer `urng_sentinel_t`

Reply:
I have a PR #1815 I'll do it in that PR.